src/test/java/edu/hm/hafner/analysis/ReportTest.java

```diff
@@ -753,6 +760,21 @@ void shouldReadIssueFromOldSerialization() {
 
         assertThatSerializableCanBeRestoredFrom(restored);
     }
+    
+    @Test
+    void shouldSerializeAndDesieralize() throws IOException, ClassNotFoundException {
```

Comment:
Actually this test is already part of the base class

Reply:
(If everything is done we still need to write the new serialization to disk and fix the corresponding test)